lib/elixir/lib/node.ex

```diff
@@ -264,4 +264,28 @@ defmodule Node do
   def get_cookie() do
     :erlang.get_cookie()
   end
+
+  @doc """
+  Loads a local, compiled module onto all connected nodes.
+  
+  Returns `{:error, :nofile}` if the object code (i.e. ".beam" file) for the module 
+  could not be found locally. 
+
+  Returns `true` if the module was successfully loaded and deployed to the 
+  connected nodes. Returns `false` otherwise.
+
+   ## Examples
+
+      iex> Node.load_module(NoSuchModuleExists)
+      {:error, :nofile}
+
+  """
+  @spec load_module(module) :: boolean | {:error, :nofile}
```

Comment:
Also, let's make it return `{:ok, boolean} | {:error, :nofile}` or `:ok | {:error, :badrpc} | {:error, :nofile}`. Thoughts?